apps/web/src/app/(main)/about/page.tsx

```diff
@@ -0,0 +1,230 @@
+import Image from 'next/image'
+import {
+  AritraImg,
+  AtanuImg,
+  RajdipImg,
+  SambitImg,
+  SawanImg,
+  SwatiImg
+} from '@public/about/team'
+import { ColorBGSVG } from '@public/hero'
+import { TracingBeam } from '@/components/ui/tracing-beam'
+
+const teamData = [
+  {
+    name: 'Sawan Bhattacharya',
+    role: 'Founder & CEO',
```

Comment:
Let's move with cofounder for both of us

Reply:
Since others have title too so we have to give this